src/lib/vpn/edit_gw_secrets.rb

```diff
@@ -192,10 +192,10 @@ def mk_gw_cert_frame
             Frame(_("Gateway certificate"), VBox(
                 Left(MinWidth(40, HBox(
                     InputField(Id(:gw_cert), _("Path to certificate file"), ""),
-                    PushButton(Id(:select_gw_cert), _("Pick.."))))),
+                    Bottom(PushButton(Id(:select_gw_cert), _("Pick..")))))),
```

Comment:
BTW I'd replace the "Pick" label with [Label.Select](https://github.com/yast/yast-yast2/blob/82c372326724b616d39b65067e01311151a2775a/library/general/src/modules/Label.rb#L174-L177) code - this would be more consistent with the other dialogs and it would save some translators work...

Reply:
Thank you @lslezak for the suggestion! I had focused on resolving what was reported in the bug. Could you please check the updated diff and review if it meets the current standard?